src/Core/Commander/Providers/Fetcher.js

```diff
@@ -0,0 +1,41 @@
+import { TextureLoader } from 'three';
```

Comment:
This is not yet the case for the moment, but a good principle will be not to contaminate our code with that of THREE.js. (Especially for loading)
The Fetcher can return an image and not a texture.